data/categories.txt

```diff
@@ -1495,7 +1495,7 @@ nl:^Computerwinkel
 fi:^Tietokonekauppa
 fr:^Magasin d'informatique
 de:^Computergeschäft
-hu:^Számítógép üzlet
+hu:^Számítógépüzlet
```

Comment:
Are you sure that we don't need the version with a space?
We currently do not insert spaces automatically.

Reply:
Yes, I am sure.
In Hungarian, if you have a composed noun which is made of two parts or more than two parts but is at most 6 sillables long, it must be written together.